buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -9,22 +9,24 @@ internal class YamlWriter {
       check(base.serviceAccountCredentials.isPresent) { "ServiceAccountCredentials in fladle extension not set. https://github.com/runningcode/fladle#serviceaccountcredentials" }
     }
     check(base.debugApk.isPresent) { "debugApk must be specified" }
-    check(base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank()) {
+    check(base.sanityRobo.get() == true || (base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank())) {
```

Comment:
If the sanityRobo is set, does the instrumentation APK also need to be present?

What is the expected behavior if both `sanityRobo` is set and `roboScript` is set?

Reply:
> If the sanityRobo is set, does the instrumentation APK also need to be present?
No, that's why I've added `base.sanityRobo.get() == true ||` which basically skip validation for sanity robo run

> What is the expected behavior if both sanityRobo is set and roboScript is set?
Same as for `instrumentationApk` and `additionalTestApks` -- it is not appended to config yml